src/utils/validateIdentifierInputs.ts

```diff
@@ -43,7 +43,7 @@ export const validateIdentifierInputs = (
     ) {
       throw new Error(ERROR_MESSAGE.INVALID_PHONE_AND_COUNTRY_CODE);
     }
-    if (!isMatchRegex(value, validationRegex)) {
+    if (!isMatchRegex(value.trim(), validationRegex)) {
       throw new Error(ERROR_MESSAGE.INVALID_IDENTIFIER_INPUT);
     }
     if (textInputType === "PHONE_NUMBER" && !fullPhoneNumberValidator(value)) {
```

Comment:
any reason why we are not calling `trim()` for the phone number?

Reply:
updated!